README.md

```diff
@@ -30,26 +30,27 @@ The `main.tf` file will configure a daemon Service within the specified ECS Clus
-| ssm_parameter_encrypted | Set this to `true` if the SSM parameter is/should be encrypted                      | `bool`        | `false`                    |    no    |
-| ssm_parameter_kms_arn   | The ARN of the KMS key to use when `ssm_parameter_encrypted` is `true`              | `string`      | `""`                       |    no    |
-| ssm_parameter_name      | The name to use for the Lacework agent access token when using SSM                  | `string`      | `"/lacework/access_token"` |    no    |
-| use_existing_iam_role   | Set this to `true` to use an existing IAM role                                      | `bool`        | `false`                    |    no    |
-| use_ssm_parameter_store | Set this to `true` to use SSM to store the Lacework agent access token              | `bool`        | `false`                    |    no    |
+| Name                         | Description                                                                         | Type          | Default                           | Required |
+| ---------------------------- | ----------------------------------------------------------------------------------- | ------------- | --------------------------------- | :------: |
+| ecs_cluster_arn              | The ARN of the ECS cluster in which to deploy the Lacework agent                    | `string`      | n/a                               |   yes    |
+| ecs_launch_type              | The desired launch type for the Lacework agent ECS Service                          | `string`      | `"EC2"`                           |    no    |
+| ecs_service_name             | The desired name for the Lacework agent ECS Service                                 | `string`      | `""`                              |    no    |
+| ecs_task_family_name         | The desired name for the Lacework agent ECS Task Definition                         | `string`      | `""`                              |    no    |
+| iam_role_arn                 | The IAM role ARN to use when `use_existing_iam_role` is `true`                      | `string`      | `""`                              |    no    |
+| iam_role_name                | The IAM role name to use when `use_existing_iam_role` is `false`                    | `string`      | `""`                              |    no    |
+| iam_role_tags                | The tags to apply to a created IAM role                                             | `map(string)` | `{}`                              |    no    |
+| lacework_access_token        | The access token for the Lacework agent                                             | `string`      | n/a                               |   yes    |
+| lacework_datacollector_image | The image used to deploy the Lacework datacollector.                                | `string`      | `"lacework/datacollector:latest"` |    no    |
```

Comment:
If the feature is for the user to specify a version/tag, should we make this input `lacework_datacollector_image_tag` and point it to `latest`?